include/openssl/types.h

```diff
@@ -227,4 +227,5 @@ typedef unsigned long ossl_uintmax_t;
 #ifdef  __cplusplus
 }
 #endif
-#endif                          /* def OPENSSL_TYPES_H */
+
+#endif /* OPENSSL_TYPES_H */
```

Comment:
Note: unfortunately, this unnecessary unrelated change caused a conflict when cherry-picking to 1.1.1. Please raise a separate pull request for 1.1.1, if you want this patch to be backported.